correction of Feature Element documentation Fixes #29 (#63)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcFeatureElementSubtraction/Documentation.md

The local placement for _IfcFeatureElementSubtraction_ is defined in its supertype _IfcProduct_. It is defined by the _IfcLocalPlacement_, which defines the local coordinate system that is referenced by all geometric representations. The local placement is always defined in relation to the local placement of the building element from which the feature element substration is substracted:

> * The _PlacementRelTo_ attribute of _IfcObjectPlacement_ shall point (if given) to the object placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute (the parent element of the feature).

Reviewer: I would make the '(if given)' consistent across Subtraction and Addition. Either have it on both or none.

Author: Sorted

> * The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point (if given) to the local placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute. (the parent element of the feature).

Reviewer 2: ```suggestion
* The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point (if given) to the local placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute (the parent element of the feature).
```

Only the main improvement here.

> * The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point (if given) to the local placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute. ( the parent element of the feature ).

Reviewer: ```suggestion
* The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point (if given) to the local placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute. (the parent element of the feature).
```

> * The _PlacementRelTo_ relationship of _IfcLocalPlacement_ shall point (if given) to the local placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute (the parent element of the feature).

Reviewer 2: ```suggestion
* The _PlacementRelTo_ attribute of _IfcObjectPlacement_ shall point (if given) to the object placement of the _IfcElement_, which is used in the _VoidsElements.RelatingElement_ inverse attribute (the parent element of the feature).
```
